jerry-core/include/jerryscript-port.h

```diff
@@ -111,27 +111,25 @@ void JERRY_ATTR_FORMAT (printf, 2, 3) jerry_port_log (jerry_log_level_t level, c
  */
 
 /**
- * Jerry time zone structure
- */
-typedef struct
-{
-  int offset;                /**< minutes from west */
-  int daylight_saving_time;  /**< daylight saving time (1 - DST applies, 0 - not on DST) */
-} jerry_time_zone_t;
-
-/**
- * Get timezone and daylight saving data
+ * Get local timezone adjustment, in milliseconds. This value, added to a timestamp in UTC
+ * time, should yield local time. Ideally, this function should satisfy the stipulations
+ * applied to LocalTZA in section 20.3.1.7 of ecmascript version 9.0.
```

Comment:
Timezone might better be time zone, but I'm not completely sure, even the spec uses timezone at least once (although it uses time zone more often). About ECMAScript instead of ecmascript, I'm positive.

And I also think that the documentation of all other port functions are kept in sync in the docs/05.PORT-API.md and in jerry-core/include/jerryscript-port.h. This should also happen here. Perhaps the two texts should be merged into a single description and used at both places. (Especially as the current text here is a little bit vague about the case when `is_utc == false`.)